### DISCUSSION TITLE
Any need for Product design contribution?

### DISCUSSION THREAD

**Discussion by Elia-Green** [anyproto/contributors#205] [Q&A]
Hi all :) I would love to contribute and wanted to ask if there is anything needed regarding user research, prototyping, and interface design.
Can share my portfolio if relevant,
Cheers

**Comment by vikasassudani** [comment]
Hey, same here. I would also love to contribute on the Product Design side of the Anytype :) 
Looking forward to it :)
Let me know if I can help and share my porttfolio too

**Comment by danitesler** [comment]
+1 to all of you

**Comment by ghost** [comment]
Just a note: could anyone offering such services make sure they're explicit about whether they're offering as a volunteer (e.g. free because they want to get involved in AnyType) or soliciting as a paid service (in which case it's spam and doesn't belong here)? It helps to know how to respond I presume.

**Comment by Katsalukha** [comment]
@Elia-Green @Vikas-404 @danylpo yep if you want to volunteer as @carrotcypher mentioned feel free to drop your portfolios at alex@anytype.io I'll be happy to check & share them with our product & design teams. Cheers!

**Comment by Konstantinos019** [comment]
Hi there, I am a product designer as well and I wanted to contribute to open source projects. I found this [platform](https://contribute.design/). Check it out if it makes sense for your needs. There are guidelines here to display the project as available for design contribution.
